qutip/tests/test_states.py

```diff
@@ -33,10 +33,10 @@
 
 import pytest
 import numpy as np
-from numpy.testing import assert_, run_module_suite
 import qutip
-from qutip import (expect, destroy, coherent, coherent_dm, thermal_dm,
-                   fock_dm, triplet_states)
+from functools import partial
+from itertools import combinations
+N = 5
```

Comment:
Is the global `N` actually used anywhere in this file? Even if the answer is yes, I'm not wild about having this global size that is shadowed by numerous local variables and pytest parameters. Could we remove it or at least give it a more specific name and perhaps move it closer to where it's used and make it specific to a specific test? Also happy with various other solutions to this.